base/src/commands.rs

```diff
@@ -1,6 +1,105 @@
-use crate::constants::{TpmCap, TpmCc, TpmPt, TpmSu};
+use crate::constants::TpmCc;
 use crate::Marshalable;
-use crate::{TpmiYesNo, TpmlDigest, TpmlPcrSelection, TpmsCapabilityData};
+
+/// [TPM2.0 1.83] 9 Start-up
```

Comment:
I have 1 two little nits,


I am not sure how would this module comment be rendered, but the way to go would be doing it something like this in `startup.rs`:

``
//! [TPM2.0 1.83] 9 Start-up

use crate::commands::{Marshalable, TpmCommand};
use crate::constants::{TpmCc, TpmSu};

/// [TPM2.0 1.83] 9.2 _TPM_Init
pub struct InitCmd {}
// .. the rest of the file
```

Reply:
Fixed